src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -382,3 +382,10 @@ def simulated_test_runner():
                 )
 
                 yield runner
+                print("\n\n\n")
+                print(
+                    f"   To replay this test for the blue team, run \n./tbots.py run thunderscope --blue_log {blue_logger.log_folder}\n    from the `src` folder.\n\n"
```

Comment:
super small nit: is it supposed to have 3 spaces at the start of this line, and 4 spaces before the "from the `src` folder" line? 
I would personally suggest combining the first and third line into one sentence, and then having the command on the second line. This way it would take slightly less space and would be less "spammy". Something like:
```
f"To replay this test for the blue team, from the `src` folder run \n./tbots.py run thunderscope --blue_log {blue_logger.log_folder}\n"
```
What do you think?

Reply:
sure